AlphaWallet/Market/Coordinators/ImportMagicLinkCoordinator.swift

```diff
@@ -291,16 +291,8 @@ class ImportMagicLinkCoordinator: Coordinator {
 
     private func handleNormalLinks(signedOrder: SignedOrder, recoverAddress: AlphaWallet.Address, contractAsAddress: AlphaWallet.Address) {
         getERC875TokenBalanceCoordinator = GetERC875BalanceCoordinator(forServer: server)
-        getERC875TokenBalanceCoordinator?.getERC875TokenBalance(for: recoverAddress, contract: contractAsAddress) { [weak self] result in
```

Comment:
Would you help to break it up, at least into a separate commit? Easier to tell what it's for.